### ISSUE TITLE
Requesting Fullscreen on iPadOS 16.5.1 Results in Undefined Exception/Error

### ISSUE THREAD

**Issue by cybereality** [mozilla-mobile/firefox-ios#15253]
## Steps to reproduce
Call `element.requestFullscreen();`

### Expected behavior
Page element goes into full screen mode.

### Actual behavior
Undefined exception/error occurs, browser does not enter fullscreen mode.

### Device & build information
* Device: ? iPad Mini (6th Gen) iPadOS 16.5.1
* Build version: ? Firefox 114.3 (31916)
* First seen version: Just noticed.
### Notes
This code works in Safari and Chrome on the same iPad. Attached a simple reproduction project. Thanks.
Attachments: 
[FirefoxFull.zip](https://github.com/mozilla-mobile/firefox-ios/files/11901230/FirefoxFull.zip)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6843)

**Comment by lmarceau** [comment]
This probably relates to https://github.com/mozilla-mobile/firefox-ios/issues/11574

**Comment by janbrasna** [comment]
The support for arbitrary elements has to be explicitly configured.

I've added the missing bits to the example above and saved that to https://jsfiddle.net/4vcf9h36 for easier access.

The current status is https://github.com/mozilla-mobile/firefox-ios/issues/30540#issuecomment-4518131091 where I'm also tracking the reasons for the exception. For the actual feature I've opened #33992 separately as there were some blockers when it launched.
